db/migrate/20240122152631_change_domain_to_url.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ChangeDomainToUrl < ActiveRecord::Migration[6.1]
+  def self.up
+    rename_column :sites, :domain, :url
```

Comment:
you can actually just use the one method here `self.change` and rails will figure out the inverse of the rename depending if you are migrating up or down on its own